stock_orderpoint_move_link/models/stock.py

```diff
@@ -27,8 +27,8 @@ def _get_stock_move_values(
             company_id,
             values,
         )
-        if "orderpoint_id" in values:
+        if "orderpoint_id" in values and values["orderpoint_id"]:
```

Comment:
I would have written something like that
```suggestion
        orderpoints = values.get("orderpoint_id") or values.get("orderpoint_ids")
        if orderpoints:
            vals["orderpoint_ids"] = [(4, orderpoint.id) for orderpoint in orderpoints]
```

Reply:
@TDu Do you want to apply this ?

Reply:
@rousseldenis The change makes sense, but it is still a cosmetic one IMO. So I will not